docs/development.rst

```diff
@@ -1,20 +1,19 @@
-=====================
```

Comment:
I believe the adornments were correct to start with, as defined by the tools that ship with docutils.
```lisp
(defcustom rst-preferred-adornments '((?= over-and-under 1)
				      (?= simple 0)
				      (?- simple 0)
				      (?~ simple 0)
				      (?+ simple 0)
				      (?` simple 0)
				      (?# simple 0)
				      (?@ simple 0))
```

Reply:
I just moved the titles levels one step down so it is coherent with the rest of the documentation.
Keeping this level would mess a bit with the other titles.

Reply:
No, I don't think it would. The rest of the documents use standard adornments. E.g., `overview.rst`:
```rest
==========
 Overview
==========
```

`api.rst`:
```rest
===============
 API Reference
===============


Protocol APIs
=============

.. automodule:: BTrees.Interfaces
```

Reply:
Cool, thanks! 

But it looks like now the commits are all confused 😭

Reply:
Are you sure? I just rebased against master.

Reply:
Ok that should be fixed now.